package-lock.json

```diff
@@ -1,9446 +0,0 @@
-{
```

Comment:
Why do we drop the package lock? Not a huge deal either way but will potentially add a bit of work if we want to keep abreast of updates coming from the mainline package

Reply:
because we typically install dependencies using yarn. so wanted to reduce risk of the two dependency trees getting out of sync.

do you mean we could more easily see where the dep trees had diverged if we had kept the package.lock? as you suggest, can regenerate in the future if we see fit... let me know if there is a more explicit 'you are diverging from the master package dependency' workflow.